openapi.yml

```diff
@@ -448,9 +448,6 @@ components:
       description: A preserved object instance
       type: object
       properties:
-        id:
-          type: integer
-          example: 123
```

Comment:
this doesn't actually get returned in the response, and i don't think consumers are actually looking for it (because it's just the internal DB PK, but everything does lookup and matching by druid in other systems)